ruff.toml

```diff
@@ -5,11 +5,13 @@ extend-select = [
 
 	# local
 	"ISC",
+	"PERF",
 	"RUF010",
 	"RUF100",
 	"UP",
 ]
 ignore = [
+	"PERF203",
```

Comment:
This exclusion needs to go below a `# local` indicator. Otherwise, it conflicts with the other ignores from jaraco/skeleton.

Reply:
Where is this documented? I have read [Config file discovery](https://docs.astral.sh/ruff/configuration/#config-file-discovery) and other parts of the documentation, but couldn't find anything using _indicator_ and _local_.

Reply:
By the way, isn't the situation the same in setuptools?
```
        "YTT", # flake8-2020
]
ignore = [
        "TRY301", # raise-within-try, it's handy
        "UP015", # redundant-open-modes, explicit is preferred
        "UP030", # temporarily disabled
        "UP031", # temporarily disabled
        "UP032", # temporarily disabled
        "UP038", # Using `X | Y` in `isinstance` call is slower and more verbose https://github.com/astral-sh/ruff/issues/7871
```